app/store/mutations.js

```diff
@@ -57,16 +58,16 @@ export const FIREBASE_INIT = (state, store) => {
             okButtonText: 'OK'
           }).then(() => {
             console.log('Alert dialog closed')
-            store.commit('SET_FOREGROUND', true)
+            store.commit(ActionTypes.SET_FOREGROUN, true)
             changeTab(tabName)
           })
         } else {
-          store.commit('SET_FOREGROUND', false)
+          store.commit(ActionTypes.SET_FOREGROUND, false)
           // cuando la aplicacion vuelve de a de background, si recibio con anterioridad
```

Comment:
We should refactor this comment like we did with the others: translate and move up.

Reply:
Yes! I didn't play attention, but i go to correct it

Reply:
This comment was translated in [this branch](https://github.com/Cambalab/ns-vue-radio/pull/88) and was merged in develop

Reply:
Nice, then solving the merge conflicts should fix it.

Reply:
Yes :blush: